tsl/test/isolation/specs/compression_conflicts_iso.spec

```
@@ -167,3 +189,7 @@ permutation "CA1" "CAc" "I1" "SChunkStat" "LockChunk1" "IBRR" "Iu1" "RC"  "Unloc
 permutation "CA1" "CAc" "I1" "SChunkStat" "LockChunk1" "IBS"  "Iu1" "RC"  "UnlockChunk" "Ic" "SH" "SA" "SChunkStat" "SU"
 permutation "CA1" "CAc" "I1" "SChunkStat" "LockChunk1" "IN1"  "RC" "UnlockChunk" "INc" "SH" "SA" "SChunkStat"
 permutation "CA1" "CAc" "I1" "SChunkStat" "LockChunk1" "INu1" "RC" "UnlockChunk" "INc" "SH" "SA" "SChunkStat" "SU"
+
+## two compression jobs should be able to work on different chunks independently 
+permutation "s1_compress_one" "s2_compress_one" "s2_commit" "s1_commit"
+permutation "s2_compress_one" "s1_compress_one" "s1_commit" "s2_commit"
```

Since the sessions are symmetric, I think it is more meaningful to compare these two:
```suggestion
permutation "s1_compress_one" "s2_compress_one" "s2_commit" "s1_commit"
permutation "s2_compress_one" "s1_compress_one" "s2_commit" "s1_commit" 
```